.github/workflows/release-check-precondition.yml

```diff
@@ -0,0 +1,39 @@
+name: Release check precondition
+
+on:
+  push:
+    branches:
+      - 'hotfix/**'
+      - 'release/**'
+    tags:
+      - '*'
+
+jobs:
+  test:
+    name: Validate pushed commit to release/hotfix or pushed tag
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Validate POM version
+        run: |
+          if [[ $GITHUB_REF =~ refs/heads/(hotfix|release)/[0-9]+\.[0-9]+\.[0-9]+.* ]]; then
+            SEM_VER_STR=${GITHUB_REF##*/}
+          elif [[ $GITHUB_REF =~ refs/tags/[0-9]+\.[0-9]+\.[0-9]+.* ]]; then
+            SEM_VER_STR=${GITHUB_REF##*/}
+          else
+            echo "Failed to parse version"
+            exit 1
+          fi
+
+          if ![[ SEM_VER_STR = `mvn help:evaluate -Dexpression=project.version -q -DforceStdout` ]]; then
```

Comment:
Have you tried, whether one can run Maven without setting up Java?

Reply:
No, I haven't tested it but if we use https://github.com/cryptomator/cryptomator/pull/2475/commits/34d6e8c4198ec54522c5c0e9241c104ddb1545c5 we won't need it anymore.

Reply:
Yes, now tested it, Maven doesn't require Java.